pkg/admission/cert/writer/internal/atomic/atomic_writer.go

```diff
@@ -0,0 +1,454 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package atomic
```

Comment:
Is this pkg copied from somewhere ? Was wondering if I can skip reviewing this ?

Reply:
Yes, it's from k8s.io/kubernetes/pkg/volume/util.
I wrote that in the 1st commit message.

Reply:
ok, sorry I missed that.